pom.xml

```diff
@@ -621,7 +621,7 @@
         <carbon.kernel.registry.imp.pkg.version.range>[1.0.1, 2.0.0)</carbon.kernel.registry.imp.pkg.version.range>
 
         <!--Carbon Identity Framework Version-->
-        <carbon.identity.framework.version>5.15.28</carbon.identity.framework.version>
+        <carbon.identity.framework.version>5.15.41-SNAPSHOT</carbon.identity.framework.version>
```

Comment:
We should not upgrade to snapshot versions. Let's get the framework PR merged and update dependency version accordingly and then get this merged

Reply:
+1. I was waiting until the framework is released and then was going to update the version.

Fixed: c4a8a4d9f812fa524742115852e7200ab9f521df